pkg/config/config_parser.go

```diff
@@ -74,6 +74,7 @@ func (p *Parser) decodeConfig(body hcl.Body, diags hcl.Diagnostics) (*Config, hc
 					config.CloudQuery.PolicyDirectory = dir
 				}
 			}
+			config.CloudQuery.SkipVersioning = viper.GetBool("skip-versioning")
```

Comment:
I think this is not required since you don't set a `viper.BindPFlag`. This flag should be also only required if you run a policy so a command argument is enough (IMO).

Reply:
agree